README.md

```diff
@@ -57,62 +69,45 @@ py -3 unrpyc.py [options] script1 script2 ...
 Options:
 ```
 $ py -3 unrpyc.py --help
-usage: unrpyc.py [-h] [-c] [-d] [-p {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15}]
-                 [-t TRANSLATION_FILE] [-T WRITE_TRANSLATION_FILE]
-                 [-l LANGUAGE] [--sl1-as-python] [--comparable] [--no-pyexpr]
-                 [--no-init-offset] [--try-harder]
-                 [--register-sl-displayable SL_CUSTOM_NAMES [SL_CUSTOM_NAMES ...]]
-                 file [file ...]
+usage: unrpyc.py [-h] [-c] [--try-harder] [-d] [-p {1,2,3 ... 15}] [-t TRANSLATE]
+[--comparable] [--no-pyexpr] [--no-init-offset]
+[--register-sl-displayable SL_CUSTOM_NAMES [SL_CUSTOM_NAMES ...]] file [file ...]
```

Comment:
This is just directly copied from the formatting of the --help output, so rewriting the indentation here is incorrect.

Reply:
Yes. A mistake.